itest/test_harness.go

```diff
@@ -250,7 +267,21 @@ func nextAvailablePort() int {
 // to each other through an in-memory gRPC connection.
 func setupHarnesses(t *testing.T, ht *harnessTest,
 	lndHarness *lntest.HarnessTest,
-	enableHashMail bool) (*tapdHarness, *serverHarness) {
+	proofCourierType proof.CourierType) (*tapdHarness,
+	*serverHarness, proof.CourierHarness) {
+
+	// If a proof courier type is specified, start test specific proof
+	// courier service and attach to test harness.
+	var proofCourier proof.CourierHarness
+	switch proofCourierType {
+	case proof.DisabledCourier:
+		// Proof courier disabled, do nothing.
+
+	case proof.ApertureCourier:
+		port := nextAvailablePort()
+		apHarness := proof.NewApertureHarness(ht.t, port)
```

Comment:
I would have expected a call to `Start()` somewhere in the code, but couldn't find it. Weird that the tests still work?